compiler/luci-interpreter/pal/linux/KernelsToBuild.lst

```diff
@@ -17,6 +17,7 @@ REGISTER_KERNEL(ExpandDims)
 REGISTER_KERNEL(Fill)
 REGISTER_KERNEL(Floor)
 REGISTER_KERNEL(FloorDiv)
+REGISTER_KERNEL(FloorMod)
```

Comment:
plz split this to another PR.
`compiler` has policy not to mix changes of `compiler/*` in one PR.